apps/server/README.MD

```diff
@@ -0,0 +1,54 @@
+# Database architecture
+
+Table Users
```

Comment:
This readme has to be rewriten according to the standarts of the team. For now that's okay, but make sure to do it before submitting the code